src/resources/common/types.ts

```diff
@@ -18,6 +18,7 @@ export interface WithConfig<T> {
 export interface AmountWithCurrency {
     amount: number;
     currency: string;
+    duration?: string;
```

Comment:
this shouldn't be here but the API returns this value, and we're using it in our code, so... 🤷‍♂️